src/mbgl/map/map.cpp

```diff
@@ -400,16 +404,19 @@ CameraOptions Map::getCameraOptions(optional<EdgeInsets> padding) const {
 }
 
 void Map::jumpTo(const CameraOptions& camera) {
```

Comment:
Yes, as mentioned in OP: `Will need to look into other bindings since they will probably expose other methods as Map::setLatLng or Map::setZoom as public API.`